spec/shoryuken/middleware/server/exponential_backoff_retry_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe Shoryuken::Middleware::Server::ExponentialBackoffRetry do
   let(:queue)     { 'default' }
-  let(:sqs_queue) { double Aws::SQS::Queue }
-  let(:sqs_msg)   { double Aws::SQS::Message, queue_url: queue, body: 'test', receipt_handle: SecureRandom.uuid,
+  let(:sqs_queue) { double Shoryuken::Queue }
+  let(:sqs_msg)   { double Shoryuken::Message, queue_url: queue, body: 'test', receipt_handle: SecureRandom.uuid,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.